src/ShopBuilder/DataFieldProvider/Item/ManufacturerDataFieldProvider.php

```diff
@@ -16,21 +16,53 @@
  */
 class ManufacturerDataFieldProvider extends DataFieldProvider
 {
-    /**
+    /** @var string */
+    public const RESPONSIBLE_NAME      = 'responsibleName';
```

Comment:
```suggestion
    private const RESPONSIBLE_NAME      = 'responsibleName';
```
These don't get used anywhere else, right?

Reply:
You are right, I will change it to private

Reply:
done